SPECS/newt/newt.spec

```diff
@@ -61,22 +75,37 @@ make %{?_smp_mflags} test
 %{_libdir}/libnewt.so
 %{_libdir}/pkgconfig/*.pc
 
+%files -n python3-newt
+%doc peanuts.py popcorn.py
+%{python3_sitearch}/*.so
+%{python3_sitearch}/*.py*
+%{python3_sitearch}/__pycache__/*.py*
+
 %changelog
-*   Wed Jun 17 2020 Joe Schmitt <joschmit@microsoft.com> 0.52.21-1
--   Update to version 0.52.21.
--   Update URL.
--   Update Source0.
--   Remove sha1 macro.
--   License verified.
-*   Sat May 09 2020 Nick Samson <nisamson@microsoft.com> 0.52.20-3
--   Added %%license line automatically
-*   Tue Sep 03 2019 Mateusz Malisz <mamalisz@microsoft.com> 0.52.20-2
--   Initial CBL-Mariner import from Photon (license: Apache2).
-*   Sat Apr 15 2017 Priyesh Padmavilasom <ppadmavilasom@vmware.com> 0.52.20-1
--   Update to 0.52.20
-*   Mon Oct 04 2016 ChangLee <changLee@vmware.com> 0.52.18-3
--   Modified %check
-*   Tue May 24 2016 Priyesh Padmavilasom <ppadmavilasom@vmware.com> 0.52.18-2
--   GA - Bump release of all rpms
-*   Tue Oct 27 2015 Mahmoud Bassiouny <mbassiouny@vmware.com>
--   Initial build.	First version
+* Tue Jul 13 2021 Muhammad Falak R Wani <mwani@microsoft.com> 0.52.21-2
+- Enable python3-newt
```

Comment:
Not sure what's the guidelines for extending disto X specs with bits from distro Y, so to be one the safe side let's also mention where we got the new bits from with something like this:

`Extending using Fedora 33 spec (license: MIT).`

Reply:
Sure :)